server.go

```diff
@@ -41,7 +41,13 @@ type errorResponse struct {
 // errorMapper maps an error from a handler into an HTTP server error.
 // TODO map other errors
 var errorMapper httprequest.ErrorMapper = func(err error) (int, interface{}) {
-	return http.StatusInternalServerError, &errorResponse{
+	status := http.StatusInternalServerError
+	cause := errgo.Cause(err)
+	switch cause {
+	case params.ErrUnparsable:
+		status = 422 // "Unprocessable Entity" - http doesn't have a const for this error.
```

Comment:
Ha, that's a new one on me. Perhaps define a constant above:

    const StatusUnprocessableEntity = 422

Or just return StatusBadRequest, I guess.